build-tools/packages/build-cli/src/commands/generate/changelog.ts

```diff
@@ -66,7 +83,14 @@ export default class GenerateChangeLogCommand extends BaseCommand<typeof Generat
 			this.error("ReleaseGroup is possibly 'undefined'");
 		}
```

Comment:
Should this now exit instead of falling through to monorepo calculation?
(Why does it say "possibly"?)

Reply:
Within a command, this.error always exits the process. This is [inherited behavior from oclif](https://oclif.io/docs/commands#thiserrormessage-string--error-options-code-string-exit-number-ref-string-suggestions-string).

Reply:
I see. It looks like `error` is properly typed as never returning. (I was assuming without the `{exit: 1 }` that it might return.
So I don't know why the follow code needs undefined checks.

And looking at the current typing I don't know why this check is even needed. `releaseGroup` should always be a defined string. (If the types are wrong, then should change those.)